cpp/tests/binaryop/unit/jit-cache-multiprocess-test.cpp

```diff
@@ -22,25 +22,32 @@
 #if defined(JITIFY_USE_CACHE)
 TEST_F(JitCacheMultiProcessTest, MultiProcessTest) {
 
-    auto tester = [&] () {
+    int num_tests = 20;
+    int *input, *output;
+    int expect = 64;
+
+    auto tester = [&] (int pid, int test_no) {
         // Brand new cache object that has nothing in in-memory cache
         cudf::jit::cudfJitCache cache;
         
-        // Single value column
-        auto column = cudf::test::column_wrapper<int>{{4,0}};
-        auto expect = cudf::test::column_wrapper<int>{{64,0}};
+        // Parent writes to output[0], child writes to output[1]
+        size_t idx = 2 * test_no + (pid == 0);
+
+        *input = 4;
+        output[idx] = 1;
 
         // make program
-        auto program = cache.getProgram("MemoryCacheTestProg", program_source);
+        auto program = cache.getProgram("MemoryCacheTestProg", program3_source);
```

Comment:
`program3_source` typo?

Reply:
No, I added another program for which the input and output are different https://github.com/rapidsai/cudf/pull/2106/files#diff-79441b75548bbc7f12c25d174ad905e9R95